chef/cookbooks/updater/recipes/default.rb

```diff
@@ -17,6 +17,21 @@
 # limitations under the License.
 #
 
+include CrowbarPacemaker::MaintenanceModeHelpers
+
+def set_maintenance_mode
+  if maintenance_mode_set_via_this_chef_run?
+    Chef::Log.info("chef-client run pid #$$ already placed this node in Pacemaker maintenance mode")
+  elsif maintenance_mode?
+    Chef::Log.info("Something else already placed this node in Pacemaker maintenance mode")
+  else
+    execute "crm --wait node maintenance" do
```

Comment:
@Itxaka No, I think it means that @vuntz needs to remember how he did it :) Or we can simply ask on `#sleha` IRC, which would be quicker.